apps/condo/domains/settings/schema/MobileFeatureConfig.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyProgressionMeterReadingsIsEnabled:Checkbox; meta:Json; ticketSubmittingIsDisabled:Checkbox'`
+ */
+
+const { Text, Relationship, Checkbox } = require('@keystonejs/fields')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+const access = require('@condo/domains/settings/access/MobileFeatureConfig')
+
+
+
+const ERRORS = {
+    TICKET_SUBMITTING_PHONES_NOT_CONFIGURED: {
+        code: BAD_USER_INPUT,
+        type: 'TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+        message: 'commonPhone field not specified',
+        messageForUser: 'api.organization.MobileFeatureConfig.TICKET_SUBMITTING_PHONES_NOT_CONFIGURED',
+    },
+    COMMON_PHONE_INVALID: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'commonPhone'],
+        type: 'COMMON_PHONE_NOT_CONFIGURED',
+        message: 'commonPhone is invalid',
+        messageForUser: 'api.organization.MobileFeatureConfig.COMMON_PHONE_INVALID',
+    },
+    ORGANIZATION_IS_REQUIRED: {
+        code: BAD_USER_INPUT,
+        variable: ['data', 'organization'],
+        type: 'ORGANIZATION_IS_REQUIRED',
+        message: 'The organization is required to create',
+        messageForUser: 'api.organization.MobileFeatureConfig.ORGANIZATION_IS_REQUIRED',
+    },
+}
+
+const MobileFeatureConfig = new GQLListSchema('MobileFeatureConfig', {
+    schemaDoc: 'Manages availability of some features in mobile application,' +
```

Comment:
It’s unclear. It assumes they you have one model per one feature.

Reply:
`MobileFeaturesConfig`?

Reply:
but google translate says that MobileFeatureConfig is ok)

Reply:
I think @pahaz means that you will not be able to customize other features using this model. See [my questions](https://github.com/open-condo-software/condo/pull/3662#pullrequestreview-1547300382) too.